numpy/distutils/mingw32ccompiler.py

```diff
@@ -124,10 +135,10 @@ def __init__ (self,
             else:
                 # gcc-4 series releases do not support -mno-cygwin option
                 self.set_executables(
-                    compiler='gcc -g -DDEBUG -DMS_WIN64 -O0 -Wall',
-                    compiler_so='gcc -g -DDEBUG -DMS_WIN64 -O0 -Wall -Wstrict-prototypes',
-                    linker_exe='gcc -g',
-                    linker_so='gcc -g -shared')
+                    compiler='gcc -march=x86-64 -mtune=generic -DMS_WIN64 -O2 -msse2 -Wall',
+                    compiler_so='gcc -march=x86-64 -mtune=generic -DMS_WIN64 -O2 -msse2 -Wall -Wstrict-prototypes',
```

Comment:
I guess the `-march` flags can go away, as the gcc-4.9 defaults should do. I will do some more testing. I will also try to remove the patch on system_info.py, as it seems to be a configuration issue. I needed this after install of `Visual C++ for Python`.